hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/TestOMRequestUtils.java

```diff
@@ -927,4 +962,11 @@ public static long addParentsToDirTable(String volumeName, String bucketName,
     }
     return parentId;
   }
+
+  public static void configureFSOptimizedPaths(Configuration conf,
+      boolean enableFileSystemPaths, String version) {
+    conf.setBoolean(OMConfigKeys.OZONE_OM_ENABLE_FILESYSTEM_PATHS,
+            enableFileSystemPaths);
+    conf.set(OMConfigKeys.OZONE_OM_LAYOUT_VERSION, version);
+  }
```

Comment:
We already defined one completely same method OzoneTestUtils#configureFSOptimizedPaths, we can reuse that and don't need to define this method again.

Reply:
Could you please go through the [comment link](https://github.com/apache/ozone/pull/1841#discussion_r563886377), here I have explained the reason. Thanks!

Reply:
On a second thought, I will retain `TestOMRequestUtils` and remove the duplicate function from `OzoneTestUtils`. I will do this change in next update.